lib/typescript/render/components/Media/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import styles from './index.module.scss';
+import {MediaHeight} from '../../providers/chatplugin/chatPluginModel';
+
+export type MediaRenderProps = {
+  height: MediaHeight;
+  contentInfo: {
+    altText: string;
+    fileUrl: string;
+    forceRefresh: boolean;
+  };
+};
+
+export const Media = ({height, contentInfo: {altText, fileUrl}}: MediaRenderProps) => (
+  <img
```

Comment:
for now we only support images but this can be customized later on with different medias